config/install.yaml

```diff
@@ -851,6 +851,14 @@ rules:
   - deployments
   verbs:
   - '*'
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
For now, we need to watch the StatefulSet of the ISBService to determine if it's done reconciling. If/when we incorporate this into the ISBService Status directly, then this can be removed.